src/types/txpool.rs

```diff
@@ -0,0 +1,204 @@
+                    }
+                },
+                "0x307e8f249bCCcfa5b245449256C5d7e6e079943e": {
+                    "3": {
+                        "blockHash": null,
+                        "blockNumber": null,
+                        "from": "0x307e8f249bcccfa5b245449256c5d7e6e079943e",
+                        "gas": "0x5208",
+                        "gasPrice": "0x2540be400",
+                        "hash": "0x54ea4d4905bf74b687ccc73e8a1fb9615357e5e82d3f716e7ab10cd8460a3221",
+                        "input": "0x",
+                        "nonce": "0x3",
+                        "to": "0x73aaf691bc33fe38f86260338ef88f9897ecaa4f",
+                        "transactionIndex": null,
+                        "value": "0x2386f26fc10000",
+                        "v": "0x1b",
+                        "r": "0xc9441e267c12a03ec2e833e21737d4bca080bc6ce13790113a207dcd06f76b5e",
+                        "s": "0x63801a9faacff49bba85c45a5b8a0e0a4f37b15240bfc19c5000be885d056e27"
+                    }
+                }
+            }
+        }"#;
+        let _: TxpoolContentInfo = serde_json::from_str(txpool_content_str).unwrap();
+    }
+
+    #[test]
+    fn should_deserialize_txpool_inspect() {
+        let txpool_inspect_str = r#"{
+            "pending": {
+                "0x0512261A7486b1e29704ac49A5EB355b6Fd86872": {
+                    "124930": "0x000000000000000000000000000000000000007E: 0 wei + 100187 gas × 20000000000 wei"
+                 },
+                "0x201354729f8d0f8b64e9a0c353c672C6a66B3857": {
+                    "252350": "0xd10e3Be2bc8f959Bc8C41CF65F60dE721cF89ADF: 0 wei + 65792 gas × 2000000000 wei",
+                    "252351": "0xd10e3Be2bc8f959Bc8C41CF65F60dE721cF89ADF: 0 wei + 65792 gas × 2000000000 wei",
+                    "252352": "0xd10e3Be2bc8f959Bc8C41CF65F60dE721cF89ADF: 0 wei + 65780 gas × 2000000000 wei",
+                    "252353": "0xd10e3Be2bc8f959Bc8C41CF65F60dE721cF89ADF: 0 wei + 65780 gas × 2000000000 wei"
+                }
+            },
+            "queued": {
+                "0x0f87ffCD71859233Eb259f42b236C8e9873444e3": {
+                    "7": "0x3479BE69e07E838D9738a301Bb0c89e8EA2Bef4a: 1000000000000000 wei + 21000 gas × 10000000000 wei",
+                    "8": "0x73Aaf691bc33fe38f86260338EF88f9897eCaa4F: 1000000000000000 wei + 21000 gas × 10000000000 wei"
+                },
+                "0x307e8f249bCCcfa5b245449256C5d7e6e079943e": {
+                    "3": "0x73Aaf691bc33fe38f86260338EF88f9897eCaa4F: 10000000000000000 wei + 21000 gas × 10000000000 wei"
+                }
+            }
+        }"#;
+        let _: TxpoolInspectInfo = serde_json::from_str(txpool_inspect_str).unwrap();
+    }
+
+    #[test]
+    fn should_deserialize_txpool_status() {
+        let txpool_status_str = r#"{
+            "pending":"0x23",
+            "queued":"0x20"
+        }"#;
+        let _: TxpoolStatus = serde_json::from_str(txpool_status_str).unwrap();
```

Comment:
Re-serialization test would be cool, after switching to `BTreeMap` you will get consistent results.